snap/snapcraft.yaml

```diff
@@ -221,12 +213,47 @@ parts:
       - libpulse0
       - libaudio2
       - libsndio7.0
-      - va-driver-all
+
+  cleanup:
```

Comment:
Should this still be named `cleanup` now that it does so much more?

Reply:
I don't care strongly enough to think of a new name.